GVFS/GVFS.Installer.Mac/CreateMacInstaller.sh

```diff
@@ -61,15 +72,18 @@ function CopyBinariesToInstall()
     removeDataDirectory="rm -Rf \"${STAGINGDIR}/${VFSFORGITDESTINATION}/Data\""
     eval $removeDataDirectory || exit 1
     
-    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/*.dylib \"${STAGINGDIR}/${PRJFSDESTINATION}/.\""
+    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/*.dylib \"${STAGINGDIR}/${VFSFORGITDESTINATION}/.\""
     eval $copyPrjFS || exit 1
     
     copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/prjfs-log \"${STAGINGDIR}/${PRJFSDESTINATION}/.\""
     eval $copyPrjFS || exit 1
     
-    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${PRJFSDESTINATION}/.\""
+    copyPrjFS="cp -Rf \"${VFS_OUTPUTDIR}/ProjFS.Mac/Native/$CONFIGURATION\"/PrjFSKext.kext \"${STAGINGDIR}/${LIBRARYEXTENSIONSDESTINATION}/.\""
     eval $copyPrjFS || exit 1
     
+    copyUnInstaller="cp -f \"${UNINSTALLERPATH}\" \"${STAGINGDIR}/usr/local/bin/.\""
```

Comment:
Is there a reason why we can't put the uninstaller in the `VFSForGit` folder? Our Git installer puts an `uninstall.sh` in the product folder.

Reply:
moved it to `/usr/local/vfsforgit`.